### PR TITLE
Provide an export option for search results

### LINKED ISSUE #779: Export search results as a spreadsheet option (similar to DL)
This came up in a discussion with Valeria and is related to issue #584 - for working with speech records it is helpful to sort the results by more than one criterion. We've decided not to provide this function before go live, but Valeria is wondering if the ability to export search results as a spreadsheet (similar to how it works in the DL) could be implemented. Then she could sort the results in the spreadsheet and figure out that way which records need which modifications.

### PR DESCRIPTION
**Pull request by aaronhelton**
Closes #779

### CODE REVIEW

**Review by jbukhari — CHANGES_REQUESTED** [review]
#1400

**Review by jbukhari — COMMENTED** [review]
Changes suggested #1439

**Review by jbukhari — APPROVED** [review]

**Comment by aaronhelton** [comment]
@jbukhari the CSV download works in the export modal now

File: dlx_rest/api/__init__.py [hunk 1]
```diff
@@ -206,6 +206,11 @@ class RecordsList(Resource):
         type=str,
         choices=['default', 'speech', 'vote', 'all', '']
     )
+    args.add_argument(
+        'of',
```
Comment (jbukhari) [inline dlx_rest/api/__init__.py:210]: Maybe "fields" be more descriptive here?

**Review by jbukhari — CHANGES_REQUESTED** [review]
#477

**Review by jbukhari — APPROVED** [review]

**Review by jbukhari — CHANGES_REQUESTED** [review]
#1478

**Review by jbukhari — CHANGES_REQUESTED** [review]
#1479

**Review by jbukhari — CHANGES_REQUESTED** [review]
#1501

**Review by jbukhari — APPROVED** [review]

**Review by jbukhari — CHANGES_REQUESTED** [review]
#1504

**Review by jbukhari — APPROVED** [review]

**Pull request merged**